### PR TITLE
Add DualMonitorTaskbar 1.22.021813

### PR DESCRIPTION
**Pull request by silverkorn**
https://sourceforge.net/projects/dualmonitortb/

A taskbar for the second monitor. 

#### Features
- Taskbar on the every monitor
- Pinned programs
- Aero support
- Window manager
- Mirror mode
- Auto-Hide
- Notification area
- Start Button (BETA)

### CODE REVIEW

File: bucket/dualmonitortaskbar.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+    "homepage": "http://sourceforge.net/projects/dualmonitortb/",
+    "version": "1.22.021813",
+    "license": {
```
Comment (Ash258) [inline bucket/dualmonitortaskbar.json]:
```suggestion
    "license": "GPL-3.0-or-later",
```

File: bucket/dualmonitortaskbar.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        "identifier": "GPL-3.0-or-later",
+        "url": "https://sourceforge.net/p/dualmonitortb/code/HEAD/tree/DualMonitorSolution/license.txt"
+    },
+    "description": "A taskbar for the second monitor.",
+    "url": "https://download.sourceforge.net/project/dualmonitortb/Releases/v01.22.021813/DualMonitor 1.22 Setup.exe",
+    "hash": "sha1:9203c5bc898d6d9c2e4febd40bc995811733bed6",
+    "innosetup": true,
+    "bin": [
```
Comment (Ash258) [inline bucket/dualmonitortaskbar.json]:
```suggestion
    "bin": "DualMonitor.exe",
```

File: bucket/dualmonitortaskbar.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
+{
+    "homepage": "http://sourceforge.net/projects/dualmonitortb/",
+    "version": "1.22.021813",
+    "license": {
+        "identifier": "GPL-3.0-or-later",
+        "url": "https://sourceforge.net/p/dualmonitortb/code/HEAD/tree/DualMonitorSolution/license.txt"
+    },
+    "description": "A taskbar for the second monitor.",
+    "url": "https://download.sourceforge.net/project/dualmonitortb/Releases/v01.22.021813/DualMonitor 1.22 Setup.exe",
```
Comment (Ash258) [inline bucket/dualmonitortaskbar.json]: Space needs to be encoded.

```suggestion
    "url": "https://download.sourceforge.net/project/dualmonitortb/Releases/v01.22.021813/DualMonitor%201.22%20Setup.exe",
```

File: bucket/dualmonitortaskbar.json [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+        "DualMonitor.exe"
+    ],
+    "shortcuts": [
+        [
+            "DualMonitor.exe",
+            "DualMonitor"
+        ]
+    ],
+    "checkver": {
+        "url": "https://sourceforge.net/projects/dualmonitortb/files/Releases/",
```
Comment (Ash258) [inline bucket/dualmonitortaskbar.json]:
```suggestion
        "url": "https://sourceforge.net/projects/dualmonitortb/files/",
```

File: bucket/dualmonitortaskbar.json [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+        "regex": "dualmonitortb/files/Releases/v0([\\d.]+)"
```
Comment (Ash258) [inline bucket/dualmonitortaskbar.json]:
```suggestion
        "regex": "﻿\\*\\*\\*\\s+Version\\s+([\\d.]+)"
```

File: bucket/dualmonitortaskbar.json [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+    },
+    "autoupdate": {
+        "url": "https://download.sourceforge.net/project/dualmonitortb/Releases/v0$version/DualMonitor $majorVersion.$minorVersion Setup.exe"
```
Comment (Ash258) [inline bucket/dualmonitortaskbar.json]:
```suggestion
        "url": "https://download.sourceforge.net/project/dualmonitortb/Releases/v0$version/DualMonitor%20$majorVersion.$minorVersion%20Setup.exe"
```

File: bucket/dualmonitortaskbar.json (hunk 1 above)
Comment (silverkorn) [reply bucket/dualmonitortaskbar.json]: Wouldn't this implies removing the `url` link to the licence?

Comment (Ash258) [reply bucket/dualmonitortaskbar.json]: It is valid SPDX identifier. There is no need for URL

**Comment by silverkorn** [comment]
@Ash258, ready for review

**Pull request merged**